src/tutorial/TutorialEventArgs.cs

```diff
@@ -87,3 +87,13 @@ public PatchEventArgs(Patch patch)
 
     public Patch Patch { get; }
 }
+
+public class ActionEventArgs : TutorialEventArgs
+{
+    public ActionEventArgs(Action data)
+    {
+        Data = data;
+    }
+
+    public Action Data { get; }
+}
```

Comment:
I'll leave this in since this probably could be useful in the future.

Reply:
Why is it Action. Data is like static data, but Action is like a basic callback / delegate type, right?

Reply:
Renamed.